src/mdnfetcher.ts

```diff
@@ -0,0 +1,44 @@
+import * as fs from "fs";
+import fetch from "node-fetch";
+import { JSDOM } from "jsdom";
+
+interface Page {
+    summary: string;
+}
+
+const knownBadDescriptions = [
+    "CustomEvent", // too vague; references truncated data
+    "IDBTransaction", // just a comment about Firefox 40
+    "RTCDtmfSender", // is just 'tbd'
+    "SVGMatrix", // too vague; references truncated data
+];
+
+fetchInterfaceDescriptions();
+
+async function fetchInterfaceDescriptions() {
+    const webIdl = require("../inputfiles/browser.webidl.preprocessed.json");
+    const interfaceNames = Object.keys(webIdl.interfaces.interface).sort();
+    const descriptions: Record<string, string> = {};
+
+    await interfaceNames.reduce(async (previousRequest, name) => {
+        // Issuing too many requests in parallel causes 504 gateway errors, so chain
+        await previousRequest;
```

Comment:
why not just make this whole thing synchronous then? (Probably because node-fetch is 500x more popular than the nearest synchronous package)